### PR TITLE
docs: add @izazw as a contributor

### PR DESCRIPTION
**Pull request by izazw**
## Description

Add izazw to the guestbook

## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @izazw! :) There are some merge conflicts on your branch that need to be solved. I highly recommend reading [Keeping Your Branch Up to Date and Handling Merge Conflicts While Waiting for PR Reviews](https://dev.to/opensauced/keeping-your-branch-up-to-date-and-handling-merge-conflicts-while-waiting-for-pr-reviews-3b3h) to help you. Also, use more descriptive titles and detailed descriptions of your proposed changes when creating PRs. It'll provide more context to your contribution, helping maintainers gain a better understanding of what you are presenting.  In this project, something like: `docs: add [GitHub username] as a contributor` would help.

**Review by adiati98 — APPROVED** [review]
Hi @izazw,
We've resolved the merge conflicts and merged your PR! 🎉

Check out [this section about how to resolve conflicts](https://github.com/open-sauced/guestbook#keeping-branch-up-to-date-and-resolving-merge-conflicts) for your future reference.

Also, in the future, please complete the PR form and describe the changes clearly. Doing so will help maintainers to review and understand the changes in your PR, and will it benefit you as well. 🙂

If you haven't, you can join our community [Discord](https://discord.com/invite/U2peSNf23P). 😊
Also, check out our [pizza-verse](https://github.com/open-sauced/pizza-verse) repo if you want to contribute further! 🍕

pizza GIF

**Pull request merged**
